.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+Thanks for contributing to Ktor's plugin registry!
+
+Before submitting your pull request, be sure to run the following:
+
+1. `./gradlew buildRegistry`: this tests and builds the expected model for the project generator using the plugin manifests
+2. `./gradlew buildTestProject`: this generates a sample project using the modified plugin files
+
+We encourage you to experiment with the different options listed here ([manifest.ktor.yaml](https://github.com/ktorio/ktor-plugin-registry/blob/main/templates/manifest.ktor.yaml)) 
```

Comment:
The link appears in parentheses. I think "in [manifest.ktor.yaml]()" should replace "here" in the sentence.